src/main/java/de/terrestris/utils/io/ZipUtils.java

```diff
@@ -51,6 +66,13 @@ public static File unzip(File zipFile, File targetDir) throws IOException {
                 Files.createDirectories(newFile.toPath());
                 if (root == null) {
                     root = newFile;
+                    if (replace) {
+                        try {
+                            FileUtils.cleanDirectory(root);
+                        } catch (Exception e) {
+                            // ignore if e.g. locked files cannot be deleted
```

Comment:
Can we have a simple log output anyway?

Reply:
Done, using slf4j for now (I can add log4j2 another time)